src/js/samples/components/SampleFilesMessage.tsx

```diff
@@ -11,28 +9,14 @@ type SampleFilesMessageProps = {
 /**
  * Displays an alert message regarding legacy sample files
  */
-export function SampleFilesMessage({ showLegacy }: SampleFilesMessageProps) {
+export default function SampleFilesMessage({ showLegacy }: SampleFilesMessageProps) {
     return showLegacy ? (
         <Alert color="orange" block>
             <p>
                 <strong>
                     Virtool now retains raw data for newly created samples instead of trimming during sample creation.
                 </strong>
             </p>
-            <p>
-                Because this is an older sample, only trimmed data is available. You can upload the original sample
-                files by dragging them onto the trimmed files they should replace.
```

Comment:
What about something like: `
Because this is an older sample, only trimmed data is available. Recreate the sample to run analysis with untrimmed reads`

@igboyes thoughts?

Reply:
Changed it to this for now